e2e/js/js-cloud-server/project.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "e2e-js-cloud-server",
+    "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+    "sourceRoot": "e2e/nextjs/app-router/src",
+    "targets": {
+        "build": {
```

Comment:
are these commands even needed? `build`, `serve`, `deploy`

Reply:
dont need build and deploy so removed those